src/main/java/gregtech/api/util/PerTickLongCounter.java

```diff
@@ -1,24 +1,28 @@
 package gregtech.api.util;
 
-import net.minecraft.world.World;
+import java.util.function.LongSupplier;
 
 public class PerTickLongCounter {
```

Comment:
I've noticed `PerTickLongCounter` is located in API package while `AveragingPerTickCounter` is not, we should probably move either one to another.

Personally I think moving `PerTickLongCounter` to non-API package is better, since it's not used in other API code and it doesn't provide a whole lot as an API.